### ISSUE TITLE
hope supporting multi-users/work profile.

### ISSUE THREAD

**Issue by QingKongBaiYu** [TigerBeanst/RnIntentClean#6]

**Comment by TigerBeanst** [comment]
It seems that the current Riru-IFW Enhancer does not support multi-users

**Comment by QingKongBaiYu** [comment]
> It seems that the current Riru-IFW Enhancer does not support multi-users

I tested it and find it's supported on my phone(A11 pixelexperience), create rules and install apk to work profile, the rules works too.

**Comment by TigerBeanst** [comment]
> > It seems that the current Riru-IFW Enhancer does not support multi-users
> 
> I tested it and find it's supported on my phone(A11 pixelexperience), create rules and install apk to work profile, the rules works too.

You means create rules manually? Can you share the IFW rule you wrote?

**Comment by QingKongBaiYu** [comment]
> > > It seems that the current Riru-IFW Enhancer does not support multi-users
> > 
> > 
> > I tested it and find it's supported on my phone(A11 pixelexperience), create rules and install apk to work profile, the rules works too.
> 
> You means create rules manually? Can you share the IFW rule you wrote?

no, install apk into main profile, create rules with Rn, install apk to work profile. the rules work in both main and work profile.

**Comment by TigerBeanst** [comment]
> > > > It seems that the current Riru-IFW Enhancer does not support multi-users
> > > 
> > > 
> > > I tested it and find it's supported on my phone(A11 pixelexperience), create rules and install apk to work profile, the rules works too.
> > 
> > 
> > You means create rules manually? Can you share the IFW rule you wrote?
> 
> no, install apk into main profile, create rules with Rn, install apk to work profile. the rules work in both main and work profile.

So what you originally meant was "Apps installed in the working profile cannot be found in the Apps list, so RnIntentClean may not support multi-users/work profile"?

**Comment by codehz** [comment]
Also if you install it in work profile, it will work fine, the only problem is only one side settings will work... (Aka, if I changed the main profile settings, the work profile settings will not working
I think they changed same file, so every time "reset rule"  will clean the rule created by opposite profile
